TYPO3.Neos/Documentation/InsideOfNeos/UserInterface/Translation.rst

```diff
@@ -2,10 +2,24 @@
 Translating the user interface
 ==============================
 
+Default Language
+================
+The `availableLanguages` are defined in `Packages/Application/TYPO3.Neos/Configuration/Settings.yaml`.
```

Comment:
needs to be two ` marks to work, see https://github.com/testbird/neos-development-collection/blob/patch-1/TYPO3.Neos/Documentation/InsideOfNeos/UserInterface/Translation.rst